src/posts/2019-01-22-why-cilium/index.md

```diff
@@ -0,0 +1,12 @@
+---
+date: '2019-01-22T17:00:00.000Z'
+title: 'Why we switched to Cilium'
```

Comment:
Can you add the company name here? That draws way more attention.

Reply:
good idea, just updated